test/Faker/Provider/pt_BR/PhoneNumberTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Test\Provider\pt_BR;
+
+use Faker\Provider;
+use Faker\Test;
+
+/**
+ * @covers \Faker\Provider\pt_BR\PhoneNumber
+ */
+final class PhoneNumberTest extends Test\TestCase
+{
+    public function testPhoneReturnsPhoneNumberWhenArgumentIsFalse(): void
+    {
+        $phoneNumber = $this->faker->phone(false);
+
+        self::assertIsString($phoneNumber);
+        self::assertNotEmpty($phoneNumber);
```

Comment:
These tests are not very useful other than letting us see whether we still trigger any deprecations here.